code/modules/cargo/console.dm

```diff
@@ -119,6 +207,18 @@
 			)
 		if((P.hidden && !(obj_flags & EMAGGED)) || (P.contraband && !contraband) || (P.special && !P.special_enabled) || P.DropPodOnly)
 			continue
+		if((P.space_gear && !space_gear))
+			continue
+		if((P.heavy_firearms && !heavy_firearms))
+			continue
+		if((P.medco_trade && !medco_trade))
+			continue
+		if((P.blackmarket && !blackmarket))
+			continue
+		if((P.animal_handing && !animal_handing))
+			continue
+		if((P.adv_sci && !adv_sci))
+			continue
```

Comment:
A simple `((cargo_flags & pack_flags) != pack_flags)` check will suffice once the bitfield variables are made.